database_interaction.py

```diff
@@ -0,0 +1,12 @@
+import User
+import json
+def write(u):
+ database_file=open(".gitignore","r+")
+ dict_n=json.load(database_file)
+ 
+ dict_n[u.email]={"name":u.name,"birth_date":u.birth_date,"email":u.email,"password":hash(u.password)}
```

Comment:
it is seems like it is not clear yet.
we'll talk about this later.
for now keep it as it is

Reply:
ok as u want